### PR TITLE
Release 1.8.0

### LINKED ISSUE #282: Release 1.8.0
### Release instructions

- [x] Branch: Starting from `develop`, cut a release branch named `release/1.8.0` for your changes.
- [x] Version bump: Bump the version number in `simple-podcasting.php`, `package-lock.json`, `package.json`, and `readme.txt` if it does not already reflect the version being released.  Update both the plugin "Version:" property and the plugin `PODCASTING_VERSION` constant in `simple-podcasting.php`.
- [x] Changelog: Add/update the changelog in both `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content. The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk`, ensuring you pull the most recent changes into `develop` first (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`). `trunk` contains the latest stable release.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] [Compare](https://github.com/10up/simple-podcasting/compare/trunk...develop) `trunk` to `develop` to ensure no additional changes were missed.
- [x] Test the pre-release ZIP locally by [downloading](https://github.com/10up/simple-podcasting/actions/workflows/build-release-zip.yml) it from the Build release zip action artifact and installing it locally. Ensure this zip has all the files we expect, that it installs and activates correctly and that all basic functionality is working.
- [x] Release: Create a [new release](https://github.com/10up/simple-podcasting/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the [closed issues on the milestone](https://github.com/10up/simple-podcasting/milestone/#?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/simple-podcasting/actions) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on [WordPress.org](https://wordpress.org/plugins/simple-podcasting/). This may take a few minutes.
- [x] Close the milestone: Edit the [milestone](https://github.com/10up/simple-podcasting/milestone/#) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.8.0` do not make it into the release, update their milestone to `1.8.1`, `1.9.0`, `2.0.0`, or `Future Release`.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Sidsector9**
Closes #282

### CODE REVIEW

**Comment by Sidsector9** [comment]
@jeffpaul there are [12 open items](https://github.com/10up/simple-podcasting/milestone/18) for the 1.8.0 milestone. Should I bump up the milestones on those and go ahead with the release?

File: package-lock.json
Comment (jeffpaul) [inline package-lock.json:1]: Given the changes here, looks like an update so will DEFINITELY want to run full regression testing on the pre-release ZIP before continuing with the release

**Review by jeffpaul — DISMISSED** [review]
We don't usually update the package-lock file during this process, but open to finding the best time/place to do that.  In this scenario, we'll want to ensure the release step for testing the pre-release ZIP is done well to ensure no regressions sneak in via dependency updates.

**Review by jeffpaul — APPROVED** [review]
Looks good, thanks for handling the release!

**Pull request merged**